src/link/README.md

```diff
@@ -2,7 +2,23 @@
 
 The `vscode-link` is a web component implementation of an [anchor element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a).
 
-## Attributes
+![Link hero](/docs/assets/images/link-hero.png)
+
+## Usage
+
+| ❌ Don't                                                                               | ✅ Do                                                                     |
+| -------------------------------------------------------------------------------------- | ------------------------------------------------------------------------- |
+| ![Link used for a refresh action](/docs/assets/images/link-dont-1.png)                 | ![Link used to navigate](/docs/assets/images/link-do-1.png)               |
+| Don't use a link for actions or commands. Try a [button](../button/README.md) instead. | Use a link to to navigate to another location in a project or on the web. |
```

Comment:
Typo: "Use a link _**to to**_ navigate to another location in a project or on the web."